cogs/startup.py

```diff
@@ -122,10 +122,9 @@ async def on_ready(self) -> None:
                     ),
                     repr(settings["MANUAL_MODERATION_WARNING_MESSAGE_LOCATION"]),
                 )
-                manual_moderation_warning_message_location_similar_to_dm: bool = (
-                    settings["MANUAL_MODERATION_WARNING_MESSAGE_LOCATION"].lower()
-                    in ("dm", "dms")
-                )
+                manual_moderation_warning_message_location_similar_to_dm: bool = settings[
+                    "MANUAL_MODERATION_WARNING_MESSAGE_LOCATION"
+                ].lower() in ("dm", "dms")
```

Comment:
Splitting mid-expression indexing harms readability. I agree that the comparison statement should be moved onto the same line, however.

Fix:
```python
manual_moderation_warning_message_location_similar_to_dm: bool = (
    settings["MANUAL_MODERATION_WARNING_MESSAGE_LOCATION"].lower() in ("dm", "dms")  # noqa: E501
)
```